### PR TITLE
Include easyTableNavigator to translation

### PR DESCRIPTION
**Pull request by CoBC**

### CODE REVIEW

**Comment by CyrilleB79** [comment]
This does not pretend to be an exhaustive review. However I have 2 remarks:
* The following file should be created: available.d/10_easyTableNavigator. For now it is missing.
* In nvdaaddons/easyTableNavigator, the stable branch should point to the last stable version, i.e. 2.2.1. More precisely master has to be merged in the stable branch.

**Comment by CoBC** [comment]
* As I can see, 10_easyTableNavigator isn't missing: https://github.com/nvaccess/mrconfig/blob/master/available.d/10_easyTableNavigator

**Comment by CyrilleB79** [comment]
> * As I can see, 10_easyTableNavigator isn't missing: https://github.com/nvaccess/mrconfig/blob/master/available.d/10_easyTableNavigator

Sorry, my bad.
Only the other point is relevant.

File: automatic.crontab [hunk 1]
```diff
@@ -62,7 +62,8 @@ poStatusHtml=/home/nvdal10n/ikiwiki/publish/poStatus.html
 02  4 * * fri $AddonTranslationUpdate debugHelper
 
 #x  5 for addons starting with e
-00  5 * * fri $AddonTranslationUpdate emoticons
+00  5 * * fri $AddonTranslationUpdate easyTableNavigator
```
Comment (feerrenrut) [inline automatic.crontab]: Please add `easyTableNavigator` at the of the 'e' section. At '20 5 * * fri $AddonTranslationUpdate easyTableNavigator`

**Review by feerrenrut — CHANGES_REQUESTED** [review]
Please confirm the following command works on the stable branch of the add-on:
`scons pot mergePot`

**Comment by CoBC** [comment]
I've made a commit on my repo to make scons pot mergePot working

**Comment by CyrilleB79** [comment]
@Oreonan, as already written before, you need to merge the `master` branch into the `stable` branch and push the `stable` branch in the `nvdaaddons` repo. Else, you won't get any translatable string.

**Comment by CoBC** [comment]
I'm waiting an invitation to nvdaaddons to do this. I requested this 
this morning to Noelia.

**Comment by CoBC** [comment]
It's now did.

**Comment by feerrenrut** [comment]
Thanks for making the updates here. Though having now been through this process with a number of other add-on's it is probably easier for me to do this manually. I'm going to close this PR and replace it with a new issue with the [Addon registration template](https://github.com/nvaccess/mrconfig/issues/new/choose): https://github.com/nvaccess/mrconfig/issues/49

**Pull request closed without merging**